src/base/projects/Stat.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  export let title: string | undefined = undefined;
+  export let ariaLabel: string | undefined = undefined;
+  export let active = false;
+  export let clickable = false;
+  export let disabled = false;
+</script>
+
+<style>
+  .stat {
+    font-family: var(--font-family-monospace);
+    font-size: var(--font-size-tiny);
+    padding: 0.5rem 0.75rem;
+    padding-bottom: 1rem; /* moving the content a tad higher to match the previous span usage */
```

Comment:
This seems a bit odd to me. Is there something off with the line-height?

Reply:
Yeah certainly, I tried to find the difference, but for some magic css reason the text content of the `span` was a few pixels higher than the one from `button`

<img width="1310" alt="Bildschirm­foto 2022-11-04 um 16 00 10" src="https://user-images.githubusercontent.com/7912302/200007103-66be3b55-c193-4c26-9e52-9fb304f8fc94.png">

If you find the reason why, please feel free to suggest any changes

Reply:
We'll never know 🤷